### PR TITLE
Trivial fixes to topology manager docs

### PR DESCRIPTION
**Pull request by markmc**

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-16663--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/topology-manager/

/lgtm

File: content/en/docs/tasks/administer-cluster/topology-manager.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ The Topology Manager currently:
 
  - Works on Nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
  - Works on Pods in the `Guaranteed` {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}
+
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/topology-manager.md]: hello @markmc . Are you able to fix a spelling typo too: s/resposible/responsible?
Line 20: `_Topology Manager_ is a Kubelet component that aims to co-ordinate the set of components that are resposible for these optimizations.`

_Topology Manager_ is a `kubelet` component that aims to coordinate the set of components that are responsible for these optimizations.

Comment (markmc) [reply content/en/docs/tasks/administer-cluster/topology-manager.md]: Well spotted, done

**Comment by kbhawkey** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**